lib/SimplePagination

```diff
@@ -0,0 +1 @@
+Subproject commit f5012a3588bed57ae60e9179a044d4c3a0be05e1
```

Comment:
Please add the code in the repository.
the simple pagination is added via submodule..